src/Sulu/Bundle/ContentBundle/DependencyInjection/Compiler/ContentImportCompilerPass.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Collects all webspace import formats.
+ */
+class ContentImportCompilerPass implements CompilerPassInterface
```

Comment:
it would make sense to extract a "general" compiler-pass for such tasks. see `Sulu/Component/Symfony/CompilerPass/TaggedServiceCollectorCompilerPass.php`